### ISSUE TITLE
Stuck at Building section

### ISSUE THREAD

**Issue by dkotrada** [paulrouget/servo-embedding-example#5]
Trying to build an example project. But stuck at this...

~~~sh
~/dev/servo/embedding-example$ cargo build --release
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating git repository `https://github.com/servo/servo`
~~~
After 10 minutes it is still there. "Updating git repository ..."

**Comment by dkotrada** [comment]
Now after long time it's finally Compiled but with Error

~~~sh
error[E0425]: cannot find function `servo_version` in module `servo`
 --> src/main.rs:4:42
  |
4 |     println!("Servo version: {}", servo::servo_version());
  |                                          ^^^^^^^^^^^^^ not found in `servo`
  |
help: possible candidate is found in another module, you can import it into scope
  |
3 | use servo::config::servo_version;
  |
~~~

**Comment by dkotrada** [comment]
This is a minimal code that I get compiled. And it runs and puts the servo version in terminal like expected

~~~cpp
extern crate servo;

use servo::config::servo_version;

fn main() {
    println!("Servo version: {}", servo_version());
}
~~~

**Comment by dkotrada** [comment]
Glutin code compiles. I can run it. Can not close with window close button.

**Comment by paulrouget** [comment]
Fixed by #6

> Can not close with window close button.

Can you file an issue for that please?
